lib/index.js

```diff
@@ -0,0 +1,41 @@
+'use babel';
+
+import { Range } from 'atom';
+import * as jsonlint from 'jsonlint';
+
+export default class LinterJsonLint {
+
+  static regex = '.+?line\\s(\\d+)';
+
+  static activate() {
+    require('atom-package-deps').install('linter-jsonlint');
```

Comment:
You can drop this parameter, it's completely optional. But you can also keep it if you want, fine either way